python/cudf/bindings/stream_compaction.pxd

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+
+# cython: profile=False
+# distutils: language = c++
+# cython: embedsignature = True
+# cython: language_level = 3
+
+from cudf.bindings.cudf_cpp cimport *
+
+cdef extern from "stream_compaction.hpp" namespace "cudf" nogil:
+
+    cdef gdf_column apply_boolean_mask(gdf_column const &input,
```

Comment:
I believe Cython only accepts `const gdf_column &input` v/s `gdf_column const &input`. Unable to find a reference for this but did a quick test locally.

Ditto below.

Reply:
Yeah, Cython is uncivilized and doesn't support East-const. Bunch of West-const hippies.

Reply:
So clearly this file wasn't being compiled by `python setup.py ...` if this is an error. so did I miss adding this file to the Cython build somewhere? Where do you define what files are compiled by Cython?

Reply:
In any case, I converted east- to west-const.

Reply:
`.pxd` files are the equivalent of headers in Cython and `.pyx` files are the equivalent of source files.

Since this `.pxd` file isn't `cimport`-ed in any of the `.pyx` files, the code isn't part of any translational unit currently.

We basically ask `setup.py` to compile all the `.pyx` files:

https://github.com/rapidsai/cudf/blob/branch-0.8/python/setup.py#L20